src/translations/en.json

```diff
@@ -92,6 +92,28 @@
         "title": "Select a card",
         "cannotMove": "Oops! It seems like you can't play. Select a Jack to scrap.",
         "resolve": "Resolve"
+      },
+      "statsScoringDialog": {
```

Comment:
This dialog isn't in the `GameView` so how about we move this whole block into the new `stats` block you've created in #727?